### ISSUE TITLE
Force rcp_log() through when it's a logging a critical error

### ISSUE THREAD

**Issue by nosegraze** [restrictcontentpro/restrict-content-pro#1766]
The logs are incredibly useful in support, but only when debug logging is actually turned on. More often than not, when someone has a problem (for example: IPN/webhook not working), they come into support but they don't have debug mode on. I sometimes have to say, _"Turn on debug mode then get back in touch when it happens gain,"_ which isn't ideal.

`rcp_log()` has a second parameter that will let us force a log through even if debug mode isn't enabled. I'd love for us to start using this if there's an actual error of some kind. Non-error messages like, _"Adding proration credits for user"_ would only show up if debugging is turned on, but it would be super helpful if things like this were always forced though:

> PayPal IPN Failed: unable to find associated member in RCP.

> Incorrect amount received in the IPN.

> The currency code in a PayPal IPN request did not match the site currency code.

> Invalid PayPal IPN attempt.

Basically anything that involves RCP not doing something it should be doing or processing some kind of critical error that we won't see somewhere else.

**Comment by tnorthcutt** [comment]
YES. Great idea.

**Comment by nosegraze** [comment]
PR: #1778
